src/org/parosproxy/paros/core/scanner/NameValuePair.java

```diff
@@ -32,6 +32,7 @@
     public static final int TYPE_COOKIE = ScannerParam.TARGET_COOKIE;
     public static final int TYPE_HEADER = ScannerParam.TARGET_HTTPHEADERS;
     public static final int TYPE_POST_DATA = ScannerParam.TARGET_POSTDATA;
+    public static final int TYPE_JSON = ScannerParam.RPC_JSON;

```

Comment:
1<<9 can be fine? And should I associate this value directly in the TYPE_JSON variable or redefine the ScannerParam.RPC_JSON one?

Reply:
Yeah, that would be fine but it would be preferable to keep them in order, e.g. use 37, after `TYPE_MULTIPART_DATA_FILE_CONTENTTYPE`.

Yes, should be defined directly in `TYPE_JSON`.